hadoop-plugin-test/src/main/gradle/positive/emergentFlow.gradle

```diff
@@ -0,0 +1,68 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath files("${project.pluginTestDir}/hadoop-plugin-${project.version}.jar", "${project.pluginTestDir}/hadoop-plugin-${project.version}-SNAPSHOT.jar")
+    // Must manually specify snakeyaml for testing, not required for users
+    classpath 'org.yaml:snakeyaml:1.18'
+  }
+}
+
+apply plugin: com.linkedin.gradle.hadoop.HadoopPlugin
+
+// Flow for testing yaml creation for emergent flows for Flow 2.0
+
+hadoop {
+  buildPath "jobs/emergentFlow"
+  cleanPath false
+
+  generateYamlOutput true
+
+  namespace('otherNamespace') {
```

Comment:
What would happen with your change if workflow is contained in the namespace block? What if namespace is included in a hadoopClosure and evalHadoopClosures is called?

Reply:
From my understanding, creating a hadoopClosure and calling evalHadoopClosures is effectively the same thing as creating a namespace.

If a workflow is defined in a namespace block then it will be included at the root of the zip if selected at zip time, then if there are .tempprops files generated in OTHER namespaces are also included in the zip then everything will be zipped together. However, if a properties file is defined in a namespace with the workflow, it will NOT create a .tempprops file because these properties are assumed to be workflow level properties (and so not shared).